Tella/Data/Networking/APICall/APIError.swift

```diff
@@ -20,6 +21,8 @@ extension APIError: LocalizedError {
             return customErrorMessage(errorCode: code)
         case .unexpectedResponse:
             return "Unexpected response from the server"
+        case .noInternetConnection:
+            return "No Internet connection. Try again when you are connected to the Internet."
```

Comment:
should we add translations for this?

cc @dhekra-rouatbi

Reply:
@ValbuenaG  added the translation